plugins/performance-lab/includes/admin/plugins.php

```diff
@@ -291,6 +291,27 @@ static function ( string $error_message ): string {
 		</div>
 		<div class="clear"></div>
 	</div>
+	<p>
+		<?php
+			$plugins_url = add_query_arg(
+				array(
+					's'             => 'WordPress Performance Team',
+					'plugin_status' => 'all',
+				),
+				admin_url( 'plugins.php' )
+			);
+			echo wp_kses(
+				sprintf(
+					/* translators: %s is the URL to the plugins screen */
+					__( 'Performance features are installed as plugins. To update features or remove them, <a href="%s">manage them on the plugins screen</a>.', 'performance-lab' ),
+					esc_url( $plugins_url )
+				),
+				array(
+					'a' => array( 'href' => true ),
+				)
+			);
+		?>
+	</p>
```

Comment:
Option 1:

[Screen recording 2024-12-13 16.27.48.webm](https://github.com/user-attachments/assets/39e01277-3081-4ab1-814d-059608444c66)

```suggestion
	<div style="text-align: center;">
		<p>
			<?php
				$plugins_url = add_query_arg(
					array(
						's'             => 'WordPress Performance Team',
						'plugin_status' => 'all',
					),
					admin_url( 'plugins.php' )
				);
				echo wp_kses(
					sprintf(
						/* translators: %s is the URL to the plugins screen */
						__( 'Performance features are installed as plugins. To update features or remove them, <a href="%s">manage them on the plugins screen</a>.', 'performance-lab' ),
						esc_url( $plugins_url )
					),
					array(
						'a' => array( 'href' => true ),
					)
				);
			?>
		</p>
	</div>
```

Reply:
I think the only part I dislike about this is that it's centered, that makes no sense to me. The rest of that screen (including the heading) is not centered - why would this need to be centered? I think it should simply be left-aligned (or right-aligned for RTL) as usual.

Then I would prefer this over the notice actually.

Reply:
I guess I just thought it looked better centered. If you want it left-aligned, I'm modifying my suggestion to make it so.

Reply:
Oh, it's already left-aligned as it is right now:

![image](https://github.com/user-attachments/assets/0d986fcd-3f09-4969-8dc4-ca813edebb90)

![image](https://github.com/user-attachments/assets/cb686b09-d681-437f-b7ee-fab618fc8d24)

![image](https://github.com/user-attachments/assets/ec732646-4c19-4274-903c-5ba7da6dec92)

Merge as you will.